src/helpers/utils.js

```diff
@@ -105,7 +109,9 @@ exports.formatHourWithMinutes = hour => (moment(hour).minutes()
 
 const roundFrenchPercentage = (number) => {
   const nf = new Intl.NumberFormat('fr-FR', { minimumFractionDigits: 2, style: 'percent' });
-  return nf.format(number);
+
+  const roundedNumber = NumbersHelper.toFixedToFloat(number, 4);
```

Comment:
pourquoi 4 et pas 2 comme c'est noté dans nf ?

Reply:
Pour les pourcentages on donne un chiffre sous la forme "0.5432" pour 54,32% par exemple. Du coup on veut recuperer quatre chiffres significatifs avant de le transformer en pourcentage, pour lequel on ne veut que deux chiffres significatifs